bodhi/metadata.py

```diff
@@ -12,23 +12,23 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-__version__ = '1.4'
+__version__ = '2.0'
 
 import os
-import glob
 import logging
 import shutil
 import tempfile
+import subprocess
```

Comment:
The actual usage of ``subprocess.check_call`` in the code here is not that bad.  It just references files that get written out to (not lots of arguments getting interpolated or anything like that).